### PR TITLE
Blog post for JuMP 1.0 release

### PR DESCRIPTION
**Pull request by odow**
Mainly an initial draft with lots of room for improvement. We should discuss this on the monthly developer call.

### CODE REVIEW

**Comment by odow** [comment]
* Miles says write for a general audience, e.g., for HN
* Have a named author(s)

**Comment by odow** [comment]
I've made some updates. Please chime in with feedback.

**Comment by mlubin** [comment]
Sorry I haven't gotten to look at this yet. I'll do a pass on Friday.

**Comment by odow** [comment]
Yeah no rush. But we're looking pretty healthy on the https://github.com/jump-dev/JuMP.jl/issues/2564 front. So once this is ready we're pretty much good to release JuMP 1.0.

File: _posts/2022-03-10-jump-1.0.md [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+* 21 Oct 2020 - [Governance structure formalized](/pages/governance/)
+* 22 Feb 2021 - [NumFOCUS signs agreement with MIT provide ongoing maintenance and support](/announcements/2021/02/22/agreement/)
+* 23 Jul 2021 - JuMP wins the [Beale--Orchard-Hays prize](http://www.mathopt.org/?nav=boh_2021) from the Mathematical Optimization Society
+* 28 Jul 2021 - [Fourth JuMP-dev workshop](/meetings/juliacon2021/)
+* 22 Oct 2021 - Third JuMP paper published: [MathOptInterface: a data structure for mathematical optimization problems](https://pubsonline.informs.org/doi/10.1287/ijoc.2021.1067)
+* 21 Feb 2022 - [NumFOCUS signs agreement with LANL to improve nonlinear support](/announcements/2021/02/22/agreement/)
+
+## Why 1.0 and how will future versions work?
+
+For some readers, reaching "1.0.0" may signal that JuMP is only now at the point
```
Comment (mlubin) [inline _posts/2022-03-10-jump-1.0.md:74]: Show not tell. I'd describe some examples in the introduction part of the blog post supporting the statement, "JuMP has been mature for a long time, and is widely used in a range of production applications". Then we don't have to repeat this here. Instead say that the 1.0 release marks the completion of the 1.0 roadmap (https://jump.dev/JuMP.jl/v0.22/developers/roadmap/#JuMP-1.0) and the transition to semantic versioning.

File: _posts/2022-03-10-jump-1.0.md [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+such as
+[power systems modeling](https://github.com/lanl-ansi/PowerModels.jl),
+[exa-scale metabolic modeling](https://github.com/LCSB-BioCore/COBREXA.jl),
+[convex programming](https://github.com/jump-dev/Convex.jl) and much more.
+
+## A brief history of JuMP
+
+* 09 Oct 2012 - First [commit](https://github.com/jump-dev/JuMP.jl/commit/688b732c13abf7182e438ca22b0b00890a08be63) to JuMP.jl repo
+* 24 Oct 2013 - [JuMP 0.1.1](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.1.1) - First public release
+* 15 Dec 2013 - [JuMP 0.1.1](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.2.0) - First support for solver callbacks
```
Comment (mlubin) [inline _posts/2022-03-10-jump-1.0.md:52]: There's nothing particularly useful at https://github.com/jump-dev/JuMP.jl/releases/tag/v0.2.0. Link to the release notes instead?

File: _posts/2022-03-10-jump-1.0.md [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+_Authors: Miles Lubin, Oscar Dowson, Iain Dunning, Joaquim Dias Garcia, Joey Huchette, and Benoît Legat._
+
+We are happy to announce the release of JuMP 1.0!
+
+Nearly 10 years in the making, the release of JuMP 1.0 represents a major
+milestone in the history of JuMP. It also represents a stable platform from
+which we can continue to build new and exciting features for a long time into
+the future.
+
+In this post we explain what "going 1.0" means for us now and what it will mean
```
Comment (mlubin) [inline _posts/2022-03-10-jump-1.0.md:18]: Somewhere in the post, we should link to the release notes and maybe summarize what's recently new with JuMP.

**Review by mlubin — APPROVED** [review]

**Comment by mlubin** [comment]
@IainNZ @joaquimg @joehuchette @blegat please approve since you're listed as co-authors :)

**Comment by IainNZ** [comment]
Exciting!

**Review by joaquimg — APPROVED** [review]

**Review by joehuchette — APPROVED** [review]
Very nice!

File: _posts/2022-03-18-jump-1.0.md [hunk 4]
```diff
@@ -0,0 +1,425 @@
...
+
+In short, semantic versioned packages have a version comprised of three numbers:
+MAJOR.MINOR.PATCH. Given a version number MAJOR.MINOR.PATCH:
+ * the MAJOR version is incremented for incompatible API changes
+ * the MINOR version is incremented when new backwards compatibile functionality
+   is added
+ * the PATCH version is incremeneted for bug fixes.
+
+By releasing JuMP 1.0.0, the core contributors are ensuring that all code you
+write using JuMP 1.0.0 will continue to work in future 1.x.y releases.
```
[the remark this answers is not in the document]
Comment (odow) [reply _posts/2022-03-18-jump-1.0.md]: How's this now?

File: _posts/2022-03-24-jump-1.0.md [hunk 5]
```diff
@@ -0,0 +1,432 @@
...
+JuMP 1.0 is not an end goal. We have big plans for the future! To get involved:
+
+ * Read our [roadmap](https://jump.dev/JuMP.jl/stable/developers/roadmap/)
+ * Join the [community forum](https://discourse.julialang.org/c/domain/opt/13)
+ * Join the [developer chatroom](https://gitter.im/JuliaOpt/JuMP-dev)
+
+## Thank you
+
+A huge number of people have contributed to packages in the JuMP-dev GitHub
+repository over the last 10 years. In order of the number of pull requests, we
```
Comment (blegat) [inline _posts/2022-03-24-jump-1.0.md:121]:
```suggestion
repositories over the last 10 years. In order of the number of pull requests, we
```

**Review by blegat — APPROVED** [review]

**Pull request merged**